boards/components/src/can.rs

```diff
@@ -0,0 +1,94 @@
+// Copyright 2022 OxidOS Automotive SRL
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+//
+// Author: Teona Severin <teona.severin@oxidos.io>
```

Comment:
Did we reach a conclusion on when / how / whether we'd like to allow inclusion of LICENSE headers, their precise format, text and/or other conventions? I recall some discussions around these issues from some time ago, and it seems that it's important to have a consistent line of argument for our license compliance here.

Reply:
We do not have any requirements here expect the request of allowing us to write the "Copyright OxidOS Automotive SRL" line.

Reply:
To document the resolution of the Core WG call of 2022-01-13, we‘re able to merge this before #3318 but would appreciate if it follows the current state of examples there, i.e. has a copyright attribution to Tock contributors as well.